### DISCUSSION TITLE
depotdownloader aint working, help?

### DISCUSSION THREAD

**Discussion by narrive** [SteamRE/DepotDownloader#364] [Q&A]
So, a few months ago I got my new PC and it doesn't decide to run DepotDownloader.
Tried to downgrade a version of Ready or Not, and it doesn't work.
https://cdn.discordapp.com/attachments/789219736781062175/1028044882444558466/Desktop_2022.10.07_-_15.41.51.02_Trim.mp4

Is it because I'm missing a few requirements? I have NET 6.0 downloaded.
Or is it just my PC that doesn't want to do anything?

**Comment by narrive** [comment]
old pc worked fine with depotdownloader, could just be windows 11

**Comment by yaakov-h** [comment]
Can you please run it from an interactive shell (cmd.exe, powershell.exe, pwsh.exe, etc.)? It seems likely that DepotDownloader or the .NET runtime host is exiting so quickly that there isn't enough time to display an output window.

If DepotDownloader _crashed_, then there may also be error events in the Windows Application Log (eventvwr.exe).
